windows/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG JENKINS_AGENT_VERSION=4.11.2-2
-ARG WINDOWS_DISTRIBUTION=           
+ARG WINDOWS_DISTRIBUTION=windowsservercore-ltsc2022

```

Comment:
I doubt this will work, from a quick look I think it's supported from kubernetes 1.23 onwards but I don't think we are running that